cassandra-four-zero-bridge/src/main/java/org/apache/cassandra/bridge/CassandraBridgeImplementation.java

```diff
@@ -176,16 +176,18 @@ public CassandraBridgeImplementation()
         kryoSerializers.put(CqlField.class, new CqlField.Serializer(cassandraTypes()));
         kryoSerializers.put(CqlTable.class, new CqlTable.Serializer(cassandraTypes()));
         kryoSerializers.put(CqlUdt.class, new CqlUdt.Serializer(cassandraTypes()));
-
-        nativeTypes = allTypes().stream().collect(Collectors.toMap(CqlField.CqlType::name, Function.identity()));
     }
 
-    @Override
     public CassandraTypes cassandraTypes()
     {
         return CassandraTypesImplementation.INSTANCE;
     }
 
+    public SparkSqlTypeConverter typeConverter()
+    {
+        return SparkSqlTypeConverterImplementation.INSTANCE;
```

Comment:
this should be a param to the bridge implementation . When Spark is needed we should have this spark instance here. but if we don't need spark, we should be able to use a different impl

Reply:
We could remove the Spark dependency from `cassandra-bridge` but it would be a lot more work and probably better to do outside this PR.